Interface/AddOns/nMinimap/tab.lua

```diff
@@ -606,8 +606,11 @@ local function BuildBNTable(total)
     wipe(BNTable)
 
     for i = 1, total do
-        local presenceID, presenceName, battleTag, _, toonName, toonID, client, isOnline, _, isAFK, isDND, _, noteText = BNGetFriendInfo(i)
-        local _, _, _, realmName, _, faction, race, class, _, zoneName, level = BNGetToonInfo(presenceID)
+		local bnetIDAccount, accountName, battleTag, isBattleTag, characterName, bnetIDGameAccount, client, isOnline, lastOnline, isAFK, isDND, messageText, noteText = BNGetFriendInfo(i)
+		
+		if (bnetIDGameAccount == nil) then return end 
+		
+		local _, _, _, realmName, realmID, faction, race, class, _, zoneName, level, gameText = BNGetGameAccountInfo(bnetIDGameAccount)
```

Comment:
It looked fine in Notepad++ but after I submitted it the spacing messed up. I changed my settings to use four spaces instead of tab so I shouldn't have any problems in the future.